include/IrrlichtDevice.h

```diff
@@ -177,6 +177,10 @@ namespace irr
 		/** \return True if window is fullscreen. */
 		virtual bool isFullscreen() const = 0;
 
+		//! Checks if the application is truly paused.
+		//! Currently, this can only happen on Android.
+		virtual bool isAppPaused() const { return false; };
```

Comment:
thinking about other platforms: what if we make this `isWindowVisible`?

Reply:
I hope that "app is paused == app is invisible" is also true in multi-window mode, but it seems to be as far as I've tested.

Will do.